### PR TITLE
add node v 14 docker images

### PR DESCRIPTION
**Pull request by danielingrey**
adding node 14

### CODE REVIEW

File: build-images/alpine-node-14/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,60 @@
+FROM node:14-alpine
+
+LABEL org.label-schema.schema-version=1.0 \
+    org.label-schema.name="Docker-Builder" \
+    org.label-schema.description="Node 10 base builder image"
+
+# Environment variavbles
```
Comment (sergio-dias-trussle) [inline build-images/alpine-node-14/Dockerfile:7]: variavbles -> variables

File: build-images/alpine-node-14/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,60 @@
+FROM node:14-alpine
+
+LABEL org.label-schema.schema-version=1.0 \
+    org.label-schema.name="Docker-Builder" \
+    org.label-schema.description="Node 10 base builder image"
```
Comment (sergio-dias-trussle) [inline build-images/alpine-node-14/Dockerfile:5]: Node 10 -> Node 14

File: build-images/debian-node-14/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,39 @@
+FROM node:14-slim
+
+# Code Climate
```
Comment (sergio-dias-trussle) [inline build-images/debian-node-14/Dockerfile:3]: No Label like above?

File: build-images/alpine-node-14/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    org.label-schema.name="Docker-Builder" \
+    org.label-schema.description="Node 14 base builder image"
+
+# Environment variables
+ENV PATH $PATH:/root/sonar-scanner/bin
+ENV CHROME_BIN /usr/bin/chromium-browser
+ENV CHROME_PATH /usr/lib/chromium/
+ENV JAVA_HOME /usr/lib/jvm/java-1.8-openjdk
+ENV NPM_VERSION 6.5.0
+ENV SONAR_SCANNER_VERSION 3.2.0.1227
```
Comment (bikash-trussle) [inline build-images/alpine-node-14/Dockerfile:13]: we dont use sonar anymore do we?

Comment (danielingrey) [reply build-images/alpine-node-14/Dockerfile:13]: nice catch, thanks

**Pull request merged**